ietf/submit/views.py

```diff
@@ -923,4 +1032,9 @@ def get_submission_or_404(submission_id, access_token=None):
     if access_token and not key_matched:
         raise Http404
 
-    return submission
\ No newline at end of file
+    return submission
+
+
+def async_poke_test(request):
```

Comment:
Vestigial?

Reply:
Nearly, but plan to hang on to it as a test point for when we deploy the async job handling to the production system.